cmd/cstor-pool-mgmt/controller/common/common.go

```diff
@@ -260,5 +261,15 @@ func Init() {
 
 	// Making RunnerVar to use RealRunner
 	pool.RunnerVar = util.RealRunner{}
+
+	// Creating maps to store imported pool info
+	if pool.ImportedCStorPools == nil {
+		pool.ImportedCStorPools = map[string]*apis.CStorPool{}
+	}
+
+	if pool.CStorZpools == nil {
```

Comment:
I know i am coming back at this after approving it earlier.
Does this name sound ok?
`CStorPoolTopologies`

Reply:
I am still trying to give a name based on if this is a map of supported topologies or something else?
Should this be then `SupportedCStorPoolTopologies`